packages/@webex/plugin-cc/src/services/core/service-utils.ts

```diff
@@ -0,0 +1,48 @@
+import * as Err from './Err';
```

Comment:
This file looks somewhat similar to utils.ts. Check if require both or if we can combine. Remove whatever is not getting used